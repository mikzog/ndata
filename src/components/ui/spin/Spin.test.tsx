import React from 'react';
import renderer from 'react-test-renderer';
import Spin from './spin';

describe('Spin', () => {
  it('renders without crashing', () => {
    const component = renderer.create(<Spin />);
    const json = component.toJSON();

    expect(json).toMatchSnapshot();
  });
});
