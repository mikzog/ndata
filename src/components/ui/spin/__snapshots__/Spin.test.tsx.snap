// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spin renders without crashing 1`] = `
<div
  className="nd-loader show"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
